apps/android/src/build.gradle

```diff
@@ -1,13 +1,15 @@
 buildscript {
```

Comment:
For my own knowledge, all of the `build.gradle`, `gradle/*`, `Bradley`, etc.. files  at `apps/android/src` are not something that RNTA provides and something we have to bring ourselves, correct?

Reply:
That is correct. @ArazAbishov probably knows the why, but my understanding is that it's just how Gradle works. I don't know if there's much we can do about that.